lib/fathead/caniuse/parse.py

```diff
@@ -31,15 +33,33 @@ def generate_answers(data):
         ])
         print ','.join(titles)
 
+        # Commented out for now -- we can revive if we ever have a way to display
+        # external links in the Related Topics Infobox
+        ## Generate Related Topics
+        #links = feature_data.get('links', [])
+        #related = u' '.join([u'[[{} {}]]'.format(link['title'], link['url']) for link in links])
+        related = ''
+
         # Generate abstract
-        abstract = feature_data['description']
+        abstract = '<p>{}</p>'.format(feature_data['description'])
+
         for browser in ['ie', 'firefox', 'chrome', 'safari', 'ios_saf', 'android']:
             agent = data['agents'][browser]
             out = browser_support(
-                browser=agent['browser'], 
-                prefix=agent['prefix'], 
+                browser=agent['browser'],
+                prefix=agent['prefix'],
                 stats=feature_data['stats'][browser])
             abstract += '<br>' + out
+
+        # Add notes to abstract, if there are any
+        notes = feature_data.get('notes', '')
+        if notes:
+            notes_in_html = markdown.markdown(notes)
+            bs = BeautifulSoup(notes_in_html)
+            for a in bs.findAll('a'):
+                a.replaceWithChildren()
+            abstract += '<p><b>Notes:</b> {}</p>'.format(bs.renderContents())
```

Comment:
Let's put `<b>Notes:<b><br>` inside the `<p>` that contains the `description`

That way we'll separate the "Notes:" text from the text above.

We should also wrap the browser support lines in one `<p>`.

The output should look like:

```html
<div class="c-info__content chomp js-ellipsis">
  
  <p>Method of positioning elements in horizontal or vertical stacks. Support includes the support for the all properties prefixed with `flex` as well as `display: flex`, `display: inline-flex`, `align-content`, `align-items`, `align-self`, `justify-content` and `order`.</p>
  
  <p>IE 11+ partial<br>Firefox 28+ ✓
    <br>Chrome 29+ ✓
    <br>Safari 9+ ✓
    <br>iOS Safari 9.0-9.2+ ✓
    <br>Android Browser 4.4.3-4.4.4+ ✓
  </p>

  <p>
    <b>Notes:</b>
    <br>Most partial support refers to supporting an older version of the specification or an older syntax.
  </p>

</div>
```

This way the output looks like:

![flexible_box_layout_module_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/17936926/690d2434-69ee-11e6-88ce-4c9619446689.png)